### ISSUE TITLE
Lab: Gradient Descent --> Train function running gives errors!

### ISSUE THREAD

**Issue by asamirr** [udacity/dlnd-issue-reports#593]
In Lesson 1: Intro to Neural Networks ---> Lab: Gradient Descent when I run the Train function, it gives the following errors even though I didn't modify it!

`/opt/conda/lib/python3.6/site-packages/ipykernel_launcher.py:13: RuntimeWarning: invalid value encountered in log`
  `del sys.path[0]`
`/opt/conda/lib/python3.6/site-packages/ipykernel_launcher.py:5: RuntimeWarning: overflow encountered in exp`
  `"""`
`/opt/conda/lib/python3.6/site-packages/ipykernel_launcher.py:13: RuntimeWarning: invalid value encountered in multiply`
  `del sys.path[0]`
`/opt/conda/lib/python3.6/site-packages/ipykernel_launcher.py:31: RuntimeWarning: invalid value encountered in greater`
`/opt/conda/lib/python3.6/site-packages/ipykernel_launcher.py:35: RuntimeWarning: invalid value encountered in double_scalars`
